app/controllers/users_controller.rb

```diff
@@ -88,4 +90,12 @@ def admin_user
   def save_avatar
     @user.avatar.attach(params[:user][:avatar]) if params[:user][:avatar]
   end
+
+  def check_empty_pw
```

Comment:
cái này devise có hỗ trợ mà
bỏ đi dc k ?

Reply:
Về phần update User, em không dùng module **Registerable** của devise mà sử dụng thủ công. Do trong app của em không cho đăng kí từ bên ngoài mà chỉ có thể tạo ra user bởi quyền admin. Do vậy không thể sử dụng việc override để hỗ trợ như phần mô tả sau đây:
https://github.com/heartcombo/devise/wiki/How-To:-Allow-users-to-edit-their-account-without-providing-a-password